protos/feast/serving/ServingService.proto

```diff
@@ -63,14 +63,21 @@ message GetFeastServingInfoResponse {
 }
 
 message FeatureReference {
-    // Project name
+    // Project name. This field is optional, if unspecified will default to 'default'.
     string project = 1;
 
     // Feature name
     string name = 2;
 
-    // Feature version
+    // Feature version. This field is optional, if unspecified will default to the
+    // feature in the latest version of the featureset.
+    // TODO: change to ignore post #676
     int32 version = 3;
+    
+    // Feature set name specifying the feature set of this referenced feature.
+    // This field is optional if the feature referenced is unique across the project
+    // in which case the feature set would be automatically infered
+    string feature_set_name = 5;
 
     // The features will be retrieved if:
     // entity_timestamp - max_age  <= event_timestamp <= entity_timestamp
```

Comment:
Is there any reason why we can't remove this max age as well? I know its not listed explicitly in your issue, but its kind of bothering me.

Reply:
I think we can remove this safely. Does not seem to be referenced anywhere. @zhilingc

Reply:
Yeah it's not used at all

Reply:
Keeping this open until its resolved.

Reply:
Corrected